src/main/java/io/github/syst3ms/skriptparser/expressions/CondExprEmpty.java

```diff
@@ -0,0 +1,63 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import java.util.Arrays;
+
+import org.jetbrains.annotations.Nullable;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.base.ConditionalExpression;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+
+/**
+ * Check if a given string or array is empty.
+ *
+ * @name Empty
+ * @pattern %strings/objects% (is|are)[1:( not|n't)] empty
+ * @since ALPHA
+ * @author Olyno
+ */
+public class CondExprEmpty extends ConditionalExpression {
+
+    static {
+        Parser.getMainRegistration().addExpression(CondExprEmpty.class,
+            Boolean.class,
+            true,
+            2,
```

Comment:
Defining the priority is redundant here with the new changes Syst3ms made.

Reply:
In fact there is no choice here. If we don't put a priority on this syntax, it creates a conflict with ``CondExprCompare``.

Reply:
It shouldn't though? Unless `empty` is some expression I'm not aware of, it should just take its time trying to parse it using CondExprCompare, but eventually parse it correctly. Though admittedly, adjusting priority manually can be done if it improves parsing speed.

Reply:
Logs if needed:
```
Exception in thread "main" io.github.syst3ms.skriptparser.parsing.SkriptParserException: Unregistered expression class : io.github.syst3ms.skriptparser.lang.VariableString
        at io.github.syst3ms.skriptparser.lang.Expression.isSingle(Expression.java:84)
        at io.github.syst3ms.skriptparser.expressions.CondExprCompare.init(CondExprCompare.java:164)
        at io.github.syst3ms.skriptparser.parsing.SyntaxParser.matchExpressionInfo(SyntaxParser.java:271)
        at io.github.syst3ms.skriptparser.parsing.SyntaxParser.parseBooleanExpression(SyntaxParser.java:189)
        at io.github.syst3ms.skriptparser.parsing.ScriptLoader.loadItems(ScriptLoader.java:104)
        at io.github.syst3ms.skriptparser.lang.SkriptEvent.loadSection(SkriptEvent.java:35)
        at io.github.syst3ms.skriptparser.lang.Trigger.loadSection(Trigger.java:31)
        at io.github.syst3ms.skriptparser.parsing.ScriptLoader.loadScript(ScriptLoader.java:76)
        at io.github.syst3ms.skriptparser.Parser.run(Parser.java:134)
        at io.github.syst3ms.skriptparser.Parser.main(Parser.java:50)
```